assets/overlays/aws-efs/patches/controller_add_driver.yaml

```diff
@@ -56,22 +74,10 @@ spec:
           env:
             - name: CSI_ENDPOINT
               value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
-            - name: AWS_ACCESS_KEY_ID
-              valueFrom:
-                secretKeyRef:
-                  name: aws-efs-cloud-credentials
-                  key: aws_access_key_id
-                  optional: true
-            - name: AWS_SECRET_ACCESS_KEY
-              valueFrom:
-                secretKeyRef:
-                  name: aws-efs-cloud-credentials
-                  key: aws_secret_access_key
-                  optional: true
             - name: AWS_SDK_LOAD_CONFIG
               value: '1'
             - name: AWS_CONFIG_FILE
-              value: /var/run/secrets/aws/credentials
+              value: /var/run/aws/auth/credentials
```

Comment:
Isn't this `AWS_CONFIG_FILE` var used by AWS SDK also in case of STS? If so it could break stuff - might be worth testing. However I broke cred. request controller when migrating the code so we might need merging this first before testing STS: https://github.com/openshift/csi-operator/pull/251

Reply:
Sounds good, let's put an hold to this PR and merge yours first.